src/transformers/default-schema-transformer.ts

```diff
@@ -125,7 +138,7 @@ function setFieldValidators(question: FormField) {
   }
 }
 
-function transformByType(question: FormField) {
+async function transformByType(question: FormField) {
```

Comment:
I actually think there's a good argument to be made to separate this out from the DefaultFormSchemaTransformer. This is inherently async whereas nothing in the default transformer otherwise is. Also, for handling attributes, it would be substantially better to batch them up into a single request... or at least to avoid re-requesting an attribute type already loaded, but that logic requires a different structure than the default class provides.